.github/workflows/test.yml

```diff
@@ -0,0 +1,34 @@
+name: test
```

Comment:
@LOUDINISouad do you understand what this file does?

Reply:
@LOUDINISouad so?

Reply:
ok, that's a good summary. github action is a tool to run CI/CD process on github servers. we will add it this week in a another PR.